test/extensions/filters/http/grpc_json_transcoder/json_transcoder_filter_test.cc

```diff
@@ -495,6 +495,12 @@ class GrpcJsonTranscoderFilterTest : public testing::Test, public GrpcJsonTransc
     ON_CALL(encoder_callbacks_, encoderBufferLimit()).WillByDefault(Return(2 << 20));
   }
 
+  virtual envoy::extensions::filters::http::grpc_json_transcoder::v3::GrpcJsonTranscoder
```

Comment:
this is not used?

Reply:
Good catch. I found a better way to do a test case with a modified proto but didn't clean up after myself. Removed.